pkg/products/cloudresources/reconciler.go

```diff
@@ -164,6 +164,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	operatorLimitRangeParams := resources.LimitRangeParams{
+		CpuRequest:    "5m",
+		MemoryRequest: "10Mi",
+		ResourceType:  corev1.LimitTypeContainer,
+	}
+	phase, err = resources.ReconcileLimitRange(ctx, client, operatorNamespace, operatorLimitRangeParams)
+	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+		events.HandleError(r.recorder, installation, phase, fmt.Sprintf("Failed to reconcile LimitRange for Namespace %s", operatorNamespace), err)
+		return phase, err
+	}
```

Comment:
Just a thought, but since this should be run on every namespace created, would it be better to move it to the `ReconcileNamespace` function instead ? 🤔 
* https://github.com/carlkyrillos/integreatly-operator/blob/e7ccc0c79e56ab343b6eb27be33727f58ad7fd4e/pkg/resources/reconciler.go#L122

Downside would be that it would be harder to find the logic / execution flow since it would be within another reconcile function, but the plus side is that this would be applied to another other namespace created in the future as long as the same function is used, assuming this is what we want 🤔

Reply:
@KevFan It's a good idea and I initially had the same thought but I ultimately opted to keep it separate from `ReconcileNamespace` because not every namespace is created through it (for example `redhat-rhoam-operator` on non-addon installs) and also for readability/traceability. So I think I'd prefer to keep them separate but I don't feel too strongly about it and can move it to `ReconcileNamespace` if you think that makes more sense.

Reply:
Cool, keeping it as is sounds good to me 👍